### PR TITLE
Serialization Docs: better intro

### PR DESCRIPTION
**Pull request by sebastianharko**
**Maybe** this is a better intro for the Serialization doc page.... 

Let me know what you think.

### CODE REVIEW

**Comment by ktoso** [comment]
PLS WHITELIST

**Comment by ktoso** [comment]
OK TO TEST

File: akka-docs/rst/java/serialization.rst [hunk 1]
```diff
@@ -5,11 +5,9 @@
  Serialization
 #####################
 
-Akka has a built-in Extension for serialization,
-and it is both possible to use the built-in serializers and to write your own.
+The messages that Akka actors send to each other are JVM objects (e.g.  instances of Scala case classes). Message passing between actors that live on the same JVM is straightforward. It is simply done via reference passing. However, messages that have to escape the JVM to reach an actor running on a different host have to undergo some form of serialization (i.e. the objects have to be converted to and from byte arrays).
 
-The serialization mechanism is both used by Akka internally to serialize messages,
-and available for ad-hoc serialization of whatever you might need it for.
+Akka itself uses Protocol Buffers to serialize internal messages (i.e. cluster gossip messages). However, the serialization mechanism in Akka allows you to write customer serializers and to define which serializer to use for what.
```
Comment (ktoso) [inline akka-docs/rst/java/serialization.rst]: customer => custom

**Review by ktoso — APPROVED** [review]
Looks good, thanks

**Comment by sebastianharko** [comment]
@ktoso fixed. thanks!

File: akka-docs/rst/java/serialization.rst [hunk 2]
```diff
@@ -5,11 +5,9 @@
  Serialization
 #####################
 
-Akka has a built-in Extension for serialization,
-and it is both possible to use the built-in serializers and to write your own.
+The messages that Akka actors send to each other are JVM objects (e.g.  instances of Scala case classes). Message passing between actors that live on the same JVM is straightforward. It is simply done via reference passing. However, messages that have to escape the JVM to reach an actor running on a different host have to undergo some form of serialization (i.e. the objects have to be converted to and from byte arrays).
```
Comment (johanandren) [inline akka-docs/rst/java/serialization.rst]: Double space after e.g.

**Review by johanandren — APPROVED** [review]
LGTM with tiny extra space issue

File: akka-docs/rst/java/serialization.rst (hunk 2 above)
Comment (johanandren) [inline akka-docs/rst/java/serialization.rst]: That was the GitHub ui misbehaving, NVM!

Comment (patriknw) [inline akka-docs/rst/java/serialization.rst]: "e.g.  instances of Scala case classes"  should not be mentioned in java docs

**Review by patriknw — COMMENTED** [review]
Thanks. Keep 'em coming. Could you sneak in the following adjustment in your other PR?

**Pull request merged**